config/101-podsecuritypolicy.yaml

```diff
@@ -16,6 +16,11 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: tekton-pipelines
+  labels:
+    app.kubernetes.io/name: psp
```

Comment:
Any reason for `psp` instead of a longer name ?

Reply:
Nope, no particular reason, do you prefer a longer name ?
I don't think anyone will need to look this up anyway, i have set a name on every resource for consistency though.
Not completely sure if making the `app.kubernetes.io/name` mandatory for every resource makes sense to be honest.

Reply:
I removed the label where it was not necessary